apps/ejabberd/src/ejabberd_listener.erl

```diff
@@ -173,13 +174,14 @@ listen_tcp(PortIPProto, Module, SockOpts, Port, IPS) ->
             ListenSocket;
         _ ->
             Res = gen_tcp:listen(Port, [binary,
+                                        {backlog, 100},
                                         {packet, 0},
                                         {active, false},
                                         {reuseaddr, true},
                                         {nodelay, true},
                                         {send_timeout, ?TCP_SEND_TIMEOUT},
                                         {keepalive, true},
-                                        {send_timeout_close, true}]),
+                                        {send_timeout_close, true} | SockOpts]),
```

Comment:
Correct me if I'm wrong, but will this prevent our overrides from ever being picked up by `gen_tcp:listen`? The 'updated' values should be in front of the proplist.